wcivf/apps/elections/helpers.py

```diff
@@ -12,10 +13,40 @@ class EEHelper:
 
     ee_cache = {}
 
+    @property
+    def base_elections_url(self):
+        """
+        Builds the URL for the elections endpoint of EveryElection API
+        """
+        return f"{settings.EE_BASE}/api/elections/"
+
+    @cached_property
+    def deleted_election_ids(self):
+        """
+        Returns a generator object that retreives ids for elections that have
+        been deleted
+        """
+        url = f"{self.base_elections_url}?deleted=1"
```

Comment:
Because we intend to run this regularly, and over time the list will get longer, it might be a good idea to also add a data filter here too. We allow filtering on [`poll_open_date`](https://github.com/DemocracyClub/EveryElection/blob/master/every_election/apps/api/views.py#L48), so adding `now() - timedelta(days=50)` (50 has no special reasoning there!) should prevent this taking longer to run over time.